src/main/resources/logback.xml

```diff
@@ -1,23 +1,24 @@
 <configuration debug="false">
   <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
-  <timestamp key="timestamp" datePattern="yyyyMMdd'T'HHmmss" />
 
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
-      <pattern>[%level] %logger{45}:%X{line} | %msg%n</pattern>
+      <pattern>%d{HH:mm:ss.SSS} [%level] %logger{45}:%X{line} | %msg%n</pattern>
```

Comment:
> [20:27:54.561](https://github.com/UCLALibrary/av-pairtree/runs/6030448227?check_suite_focus=true#step:8:95)

1. I think we also need date information here, not just time.
2. Can we also log the UTC offset or timezone to improve human readability?
3. Also for consideration, I think it would be easier to read formatted as 12-hour + AM/PM instead of 24-hour.

Reply:
If you think 24-hour is better, I can roll with it, but I do feel strongly about the first two.

Reply:
Yes, was updating Hauth's too and came to same conclusion about date being important for console log. Adding it. I'll look into UTC or timezone.

Reply:
Nos. 1 and 2 are done now (with no. 2 being configurable).

I think the 12 hr format is arguably easier for a person from the U.S. to read, but I think it's better to stick with the 24 hour format because we'd like, in the future, to be able to aggregate (and search/filter) logs. The 24 hour format is more widely used.

Reply:
Thank you!